CHANGELOG.rst

```diff
@@ -5,6 +5,13 @@ Changelog
 dev-version
 ===========
 
+Breaking Changes
+----------------
+- Removed the ``HealVis`` wrapper. Use ``pyuvsim`` instead.
+Changed
+-------
+- Updated ``PolyBeam`` to use future array shapes.
```

Comment:
I'd maybe also add that the package as a whole has been updated to use future array shapes.

Reply:
Done.